vertx-axle-clients/vertx-axle-generator/src/test/java/io/vertx/core/streams/package-info.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2011-2017 Contributors to the Eclipse Foundation
```

Comment:
Not to be fixed here, but something to update in codegen.

Reply:
those are test classes and are copied this way.

I think we should add this repository in .genignore and remove the classes (in another PR)